src/friendly-iframe-embed.js

```diff
@@ -852,8 +852,14 @@ function installPolyfillsInChildWindow(parentWin, childWin) {
   // The anonymous class parameter allows us to detect native classes vs
   // transpiled classes.
   installCustomElements(childWin, class {});
-  // eslint-disable-next-line no-undef
-  if (INTERSECTION_OBSERVER_POLYFILL || getMode().localDev || getMode().test) {
+  if (
+    // eslint-disable-next-line no-undef
+    INTERSECTION_OBSERVER_POLYFILL ||
+    // eslint-disable-next-line no-undef
+    INTERSECTION_OBSERVER_POLYFILL_INABOX ||
```

Comment:
Does `INTERSECTION_OBSERVER_POLYFILL_INABOX` affect the FIE case?

Reply:
Good catch. Not really. These use cases do not overlap. So I'm going to take it out to avoid confusion.